### PR TITLE
Resolved issue of app hanging in lower end phones

### PR DESCRIPTION
**Pull request by Thejas775**
## 📋 Description

Resolved issue of app hanging and crashing in lower end phones

### CODE REVIEW

**Comment by Thejas775** [comment]
@NikhilFegade

File: app/src/main/java/org/piramalswasthya/cho/ui/register_patient_activity/patient_details/PatientDetailsFragment.kt [hunk 1]
```diff
@@ -122,13 +122,37 @@ class PatientDetailsFragment : Fragment() , NavigationAdapter {
             dialog = builder.create()
             dialog.show()
 
+//            lifecycleScope.launch(Dispatchers.IO) {
```
Comment (drtechie) [inline app/src/main/java/org/piramalswasthya/cho/ui/register_patient_activity/patient_details/PatientDetailsFragment.kt:125]: Please don't commit commented out code.

**Comment by drtechie** [comment]
Code doesn't look DRY.

File: app/src/main/java/org/piramalswasthya/cho/ui/register_patient_activity/patient_details/PatientDetailsFragment.kt (hunk 1 above)
Comment (Thejas775) [reply app/src/main/java/org/piramalswasthya/cho/ui/register_patient_activity/patient_details/PatientDetailsFragment.kt:125]: Okay

File: app/src/main/java/org/piramalswasthya/cho/ui/commons/personal_details/PersonalDetailsFragment.kt [hunk 2]
```diff
@@ -490,7 +506,7 @@ class PersonalDetailsFragment : Fragment() {
         val patients = withContext(Dispatchers.IO) {
             patientDao.getAllPatients()
         }
-        withContext(Dispatchers.Default) {
+        withContext(Dispatchers.Main) {
```
Comment (NikhilFegade) [inline app/src/main/java/org/piramalswasthya/cho/ui/commons/personal_details/PersonalDetailsFragment.kt:509]: I think there is no need to Dispatchers.Main

**Review by NikhilFegade — CHANGES_REQUESTED** [review]

**Pull request closed without merging**